pkg/adaptor/server.go

```diff
@@ -0,0 +1,141 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package adaptor
+
+import (
+	"context"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/confidential-containers/cloud-api-adaptor/pkg/adaptor/cloud"
+	"github.com/confidential-containers/cloud-api-adaptor/pkg/adaptor/proxy"
+	"github.com/confidential-containers/cloud-api-adaptor/pkg/adaptor/vminfo"
+	"github.com/confidential-containers/cloud-api-adaptor/pkg/podnetwork"
+
+	"github.com/containerd/ttrpc"
+	pbHypervisor "github.com/kata-containers/kata-containers/src/runtime/protocols/hypervisor"
+
+	pbPodVMInfo "github.com/confidential-containers/cloud-api-adaptor/proto/podvminfo"
```

Comment:
nit: This should be grouped with the other `github.com/confidential-containers/cloud-api-adaptor` imports right?

Reply:
done